Lesson08-HW/src/app/pages/Post.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import PostProfile from "../components/Post";
+
+const Post = props => {
+  const [statePost, setStatePost] = useState({post: null});
+  useEffect(
+    axios.get("http://jsonplaceholder.typicode.com/posts/" + props.match.params.postId).then(response => setStatePost(response.data))
```

Comment:
```suggestion
    () => {axios.get("http://jsonplaceholder.typicode.com/posts/" + props.match.params.postId).then(response => setStatePost({post: response.data}))}
```
в качестве аргумента надо передавать функцию, а у вас результат выполнения `axios.get()`